### PR TITLE
Final work

### PR DESCRIPTION
**Pull request by Elisha-Tetteyfio**
In this project, I and @bruk19 created the recipe app project with the following
- Login page and registration page:
-- Should be built with Devise.
- Food list:
-- Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also quantity of a given food).
--Should lead to a form that allows users to add new food.
- Recipes list:
-- Should display a list of recipes created by the logged-in user as in the wireframe.
-- Should lead to recipe details.
-- If the user is the owner of the recipe, should allow the user to delete it.
- Public recipe list:
-- Should display a list of all public recipes ordered by newest as in the wireframe.
-- Should lead to recipe details.
- Recipe details:
-- If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
-- If the user is the owner of the recipe, should lead to the form that allows the user to add new food.
- General shopping list view (only for a group that has less than 3 members):
-- Should show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the general food list of that user).
-- Should count the total food items and total price of the missing food.
- Make sure there are no N+1 queries happening.
- Create a navigation menu that allows users to open all of the pages you created.

### CODE REVIEW

File: app/views/recipes/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.label :preparation_time, style: "display: block" %>
```
Comment (OLIPLICHE) [inline app/views/recipes/_form.html.erb:20]: - Please. On the recipe show page, you should be able to add ingredient.
- On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url `/general_shopping_list`On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url /general_shopping_list
- See  Your page

 
Capture
- **Original page**
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
# Highlights
- App working fine (✅)) 👏
- README file is descriptive 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch)  👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/recipes/_form.html.erb (hunk 1 above)
Comment (OLIPLICHE) [reply app/views/recipes/_form.html.erb:20]: - **Test**
- 
Capture

**Review by mukhammadrizooff — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive `PR` description and `Commit` history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**